sphinx/builders/linkcheck.py

```diff
@@ -42,6 +42,7 @@
 
 Hyperlink = NamedTuple('Hyperlink', (('next_check', float),
                                      ('uri', Optional[str]),
+                                     ('anchors', Set[str]),
```

Comment:
Unfortunately, this may not work well because the location info (pair of docname and lineno) become multiple for multiple anchors.

Reply:
To support multiple pairs of `(docname, lineno)`, it would be better to pass a tuple `(uri_without_anchor, list of hyperlinks)` to wqueue.